Makefile

```diff
@@ -202,6 +202,10 @@ revendor:
 clean:
 	@hack/clean.sh ./cmd/... ./extensions/... ./pkg/... ./plugin/... ./test/...
 
+.PHONY: add-license-headers
+add-license-headers: $(GO_ADD_LICENSE)
+	@./hack/add-license-header.sh
```

Comment:
IMO this doc is fine, the statement here says to run make verify before filing PR which checks linting, formatting etc.
Missing licenses can be considered linting issue and it will get checked in `make verify` step.